explore-analyze/visualize/text-panels.md

```diff
@@ -12,6 +12,9 @@ products:
 
 To provide context to your dashboard panels, add **Text** panels that display important information, instructions, images, and more. You can create **Text** panels using GitHub-flavored Markdown text.
 
+:::::{applies-switch}
+
+::::{applies-item} stack: ga 9.0
```

Comment:
The 9.2/serverless tab should be before the 9.0 one, so that the latest is displayed by default.